src/mgmt/srv_drpc.c

```diff
@@ -1568,6 +1568,45 @@ pool_query_free_tier_stats(Mgmt__PoolQueryResp *resp)
 	resp->n_tier_stats = 0;
 }
 
+static char *
+rank_ranges_str(d_rank_range_list_t *list, bool *truncated)
```

Comment:
I wonder if this should be a utility function in gurt? I don't feel strongly either way, but seems like there could be other use cases where this might be handy (e.g. daos tool).